MaxMind.MinFraud/Request/CreditCard.cs

```diff
@@ -42,6 +42,8 @@ public sealed class CreditCard
         ///  as provided by the payment processor.</param>
         /// <param name="token">A token uniquely identifying the card. This
         /// should not be the actual credit card number.</param>
+        /// <param name="was3dSecureSuccessful">Whether or not the 3DS check was
```

Comment:
I suggest adding the reasons to not include this param, eg 

> Whether the outcome of 3-D Secure verification (e.g. Safekey,
SecureCode, Verified by Visa) was successful. +true+ if customer
verification was successful, or +false+ if the customer failed
verification. If 3-D Secure verification was not used, was unavailable,
or resulted in an outcome other than success or failure, do not
include this parameter.